modules/lib/src/main/resources/assets/styles/inputtype/selector/image/image-editor.less

```diff
@@ -1,6 +1,17 @@
 .image-editor {
   .notSelectable();
 
+  &._360-540 {
+    .apply-button {
+      padding-left: 6px;
+      padding-right: 6px;
+    }
+    
+    .reset-button {
+      padding: 0px;
+    }
+  }
```

Comment:
Can you move this to the end of the class? We usually have responsive styling at the bottom.